selenium/src/test/java/za/co/absa/enceladus/selenium/AppTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * <!--
+ *   ~ Copyright 2018 ABSA Group Limited
```

Comment:
The Copyright should only have text in it, not the xml formatted comment prefix per line. (Your IntelliJ settings might be a bit off)